src/SuperSocket.Quic.Connection/QuicPipeStream.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.IO;
+using System.Net.Quic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SuperSocket.Quic.Connection;
+
+#pragma warning disable CA2252
+
+public sealed class QuicPipeStream : Stream
+{
+    private Stream _stream;
+
+    private readonly QuicConnection _connection;
+
+    public QuicPipeStream(QuicConnection connection)
+    {
+        _connection = connection;
+    }
+
+    public override bool CanRead => _stream.CanRead;
+    public override bool CanSeek => _stream.CanSeek;
+    public override bool CanWrite => _stream.CanWrite;
+    public override long Length => _stream.Length;
+
+    public override long Position
+    {
+        get => _stream.Position;
+        set => _stream.Position = value;
+    }
+
+    public async ValueTask OpenStreamAsync(CancellationToken cancellationToken)
+    {
+        _stream = await _connection.AcceptInboundStreamAsync(cancellationToken);
```

Comment:
It's inbound stream, do we need handle the outbound stream?

Reply:
The Inbound Stream seems to be all that's needed on the server